pylint/checkers/imports.py

```diff
@@ -712,6 +717,20 @@ def _check_deprecated_module(self, node, mod_path):
             if mod_path == mod_name or mod_path.startswith(mod_name + '.'):
                 self.add_message('deprecated-module', node=node, args=mod_path)
 
+    def _check_import_as_rename(self, node):
+        names = node.names
+        if not all(names[0]):
+            return
+
+        real_name = names[0][0]
+        packages = real_name.rsplit('.', 1)
+        real_name = packages[1] if len(packages) == 2 else packages[0]
+        imported_name = names[0][1]
+        if real_name == imported_name:
+            msg = "Import alias does not rename '%s' and can be rewritten." % (imported_name)
```

Comment:
I find the message a bit too verbose. What about just keeping this one (`Import alias does not...`) instead of adding it to the other part of the message?